test/integration/singlecluster/webhook/core/workload_test.go

```diff
@@ -966,5 +967,27 @@ var _ = ginkgo.Describe("Workload validating webhook", func() {
 				g.Expect(k8sClient.Status().Update(ctx, wl)).Should(testing.BeForbiddenError())
 			}, util.Timeout, util.Interval).Should(gomega.Succeed())
 		})
+
+		ginkgo.It("Should allow workload podSets count update when DynamicallySizedJob feature gate is enabled", func() {
+			features.SetFeatureGateDuringTest(ginkgo.GinkgoTB(), features.DynamicallySizedJob, true)
```

Comment:
I think we also need code to disable the gate.

Reply:
What I mean is that we need to disable the feature gate at the end of the test so that the downstream tests are not affected by the enablement. Typically what we do is a separate Describe for a new feature where the feature gate is enabled in Before each and disabled in After each. Ideally I would follow the pattern here too